CHANGELOG.md

```diff
@@ -195,12 +195,12 @@ Users/clients that do not make use of these endpoints will not be impacted.
     after receiving a 404 Not Found Response Error from the `cumulus-api`.
 - **CUMULUS-3165**
   - Update example/cumulus-tf/orca.tf to use orca v6.0.3
-
 - **CUMULUS-3215**
   - Create reconciliation reports will properly throw errors and set the async
     operation status correctly to failed if there is an error.
   - Knex calls relating to reconciliation reports will retry if there is a
     connection terminated unexpectedly error
+  - Set default async_operation_image_version to 47
```

Comment:
This is for our example deployment, so update to something like:
set default example/cumulus-tf default async_operation_image_version to 47

Also add "Notable Change" section:
The async_operation_image property of cumulus module should be updated to pull the ECR image for cumuluss/async-operation:47

Reply:
But there is only logging update.  Is it really necessary to create a new image?

Reply:
https://github.com/nasa/cumulus/pull/3372/commits/d5cc9113c9b1ff815cf34d89db0fdabd15a4bcd7

Reply:
> But there is only logging update. Is it really necessary to create a new image?

I already created the new image. Did you want me to revert? I think the it an improvement over the last since the previous version has inaccurate logging. But I'm open to reverting/deleting the newly pushed images

Reply:
ok.  Maybe update the changelog to indicate it's an logging improvement.